pkg/tnf/reel/reel.go

```diff
@@ -199,7 +221,8 @@ func (r *Reel) Step(step *Step, handler Handler) error {
 				} else {
 					before = ""
 				}
-				step = handler.ReelMatch(r.stripEmulatedRegularExpression(firstMatch), before, match)
+				amatch := r.stripEmulatedRegularExpression(firstMatch)
```

Comment:
I got a bit confused by firstMatch, amatch, match. Took me a while to realize firstMatch and amatch are regex patterns. Maybe name them differently to reflect that?

Reply:
fixed